docs/bokeh/source/docs/user_guide/topics/stats.rst

```diff
@@ -13,6 +13,13 @@ Use |quad| glyphs to create a histogram plotted from ``np.histogram`` output
 .. bokeh-plot:: __REPO__/examples/topics/stats/histogram.py
     :source-position: above
 
+.. _ug_topics_stats_pyramid:
+
+Horizontal histograms can also be plotted using the |hbar| glyphs:
```

Comment:
Let's be careful not to call the *plot* a histogram, exactly. AFAICT these are usually referred to as bar charts. Maybe something like:

> A pyramid plot is a divergent bar plot that can be used to compare histogram data from two populations. In Bokeh they can be created using |hbar| glyphs.

Reply:
Thanks.